### ISSUE TITLE
User tagging in replies causes overflow scrollbar jitter in edge cases

### ISSUE THREAD

**Issue by Juicysteak117** [bluesky-social/social-app#2388]
**Describe the bug**
When attempting to tag a user using an `@` in a reply, the user field can cause an overflow jitter in cases where the post being replied to is close to an overflow. The user field appears to briefly expand the composer which causes the overflow before resettling into the non-overflow state, causing a jitter with every update of the user field.

**To Reproduce**

1. Find an edge case (I found https://bsky.app/profile/flightforall.bsky.social/post/3khs6sp2h5b2q)
2. Draft a reply
3. Press `@`
4. Type any letters
5. Observe

**Expected behavior**

I expect this not to jitter.

**Screenshots**

Bad video that dropped frames when compressing it, but the effect should still be visible.

https://github.com/bluesky-social/social-app/assets/20595808/a59a08b8-1ee6-4b97-b11c-89396503e57e

**Details**

 - Platform: Chromium desktop
 - Platform version:
 - App version:

**Additional context**

**Comment by haileyok** [comment]
This seems fixed now I think. Any sense of if it really is?

**Comment by Juicysteak117** [comment]
Yeah I can't repro anymore. Not sure when I last could anyways because I forgot this was an issue. Fixed yay.
